lists/broken

```diff
@@ -67,6 +69,7 @@ ostinato
 owtf
 packerid # #2620, #3572, deprecated, better delete
 packet-o-matic
+pacu # missing dependencies
```

Comment:
the broken list is more for unfixable packages

for pacu (https://github.com/BlackArch/blackarch/issues/2759#issuecomment-2178714862) it's just that the dependencies in the package are longer matching upstream, and as there are a lot of them and they are changing frequently, the solution would be to change the packing to a virtual-env.

We are no putting package in the broken list before we tried to fix them.

Reply:
Here the PR for pacu  https://github.com/BlackArch/blackarch/pull/4627